snippets/sdn-deprecation-statement-managed.adoc

```diff
@@ -0,0 +1,29 @@
+// Text snippet included in the following assemblies:
+//
+// * networking/about-networking.adoc
+// * networking/openshift_sdn/assigning-egress-ips.adoc
+// * networking/openshift_sdn/editing-egress-firewall.adoc
+// * networking/openshift_sdn/enabling-multicast.adoc
+// * networking/openshift_sdn/migrate-to-openshift-sdn.adoc
+// * networking/openshift_sdn/multitenant-isolation.adoc
+// * networking/openshift_sdn/removing-egress-firewall.adoc
+// * networking/ovn_kubernetes_network_provider/rollback-to-openshift-sdn.adoc
+// * networking/ovn_kubernetes_network_provider/viewing-egress-firewall-ovn.adoc
+// * networking/openshift_sdn/about-openshift-sdn.adoc
+// * networking/openshift_sdn/rollback-to-ovn-kubernetes.adoc
+// * networking/openshift_sdn/configuring-egress-firewall.adoc
+//
+// Text snippet included in the following modules:
+//
+// * modules/nw-networking-glossary-terms.adoc
+// * modules/nw-ovn-kubernetes-migration-about.adoc
+// * modules/optimizing-mtu-networking.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+[NOTE]
+====
+OpenShift SDN CNI is deprecated as of OpenShift version 4.14. As an alternative to OpenShift SDN CNI, you can use OVN Kubernetes CNI instead.
+
+Because {product-title} follows the life cycle of SDN according to OpenShift Core Platform, SDN is deprecated for clusters using version 4.14. Clusters using version 4.15 are unable to install the SDN plug-in. In OpenShift version 4.16, you cannot upgrade your clusters if it is using the SDN plug-in. The SDN plug-in will be discontinued in OpenShift version 4.17.
```

Comment:
```suggestion
Because {product-title} follows the life cycle of SDN according to OpenShift Core Platform, SDN is deprecated for clusters as of OpenShift version 4.14. 

* Clusters upgraded to OpenShift 4.12-4.15
** Clusters use the OpenShiftSDN plugin after they are upgraded to OpenShift version 4.11 and above.
** Clusters can be upgraded to OpenShift versions up to version 4.15.
** There will soon be a migration tool to migrate from OpenShift SDN to OVN for OpenShift clusters version 4.15 and later. For more information, see [link something useful].
* Clusters running on OpenShift 4.16 and later
** Clusters using OpenShift version 4.16 and later cannot upgrade if the clusters are using the SDN plugin. 
** The SDN plugin will be discontinued in OpenShift version 4.17.
```
Or something of this sort. I don't really care if it is a note, a paragraph, or a table. But I think something that spells it all out for the customers based on their type of cluster will be useful. And we should probably run that part by Bala again, so we know it is 100% accurate.